src/app/auth/signup/page.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import * as React from 'react';
+
+import { Prisma } from '@prisma/client';
+
+import { createUserAction } from '@/app/actions/action';
+import SignUpForm from '@/components/Signup/Form';
+import toast from '@/lib/toast';
+
+const SignUp = () => {
+  const handleFormSubmit = async (payload: Prisma.UserCreateInput) => {
+    try {
+      await createUserAction(payload);
+      toast('User is successfully created. Please log in', 'success');
```

Comment:
Revisit the messages based on earlier comments 😅 

```suggestion
      toast('The user account has been successfully created. You can now log in.', 'success');
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/65/commits/56d11cf02f9d961710f30a2886e426e0e65af976